ml-kem/src/lib.rs

```diff
@@ -21,12 +21,18 @@
 //! computers.
 //!
 //! ```
-//! # use ml_kem::*;
-//! # use ::kem::{Decapsulate, Encapsulate};
-//! let mut rng = rand::rng();
+//! use ml_kem::{
+//!     MlKem768Params,
+//!     kem::{Decapsulate, Encapsulate, KeyInit}
+//! };
+//!
+//! type DecapsulationKey = ml_kem::kem::DecapsulationKey<MlKem768Params>;
```

Comment:
Perhaps we could put these type aliases together for people so they don't have to assemble them like this?

```suggestion
//! use ml_kem::ml_kem_768::DecapsulationKey;
```